src/main/clojure/clara/rules/platform.cljc

```diff
@@ -75,4 +75,10 @@
            ~@body
            (finally
              ~@(for [[tl] binding-pairs]
-                 `(.remove ~tl)))))))
\ No newline at end of file
+                 `(.remove ~tl)))))))
+
+#?(:clj
```

Comment:
I'd have thought this would be a problem since ClojureScript macros are evaluated by the JVM compiler, at least if you're not using self-hosted ClojureScript; I told @ris5438 to do it this way.  Perhaps I'm incorrect or out of date though.  Do you know offhand of any good references, blog posts, etc. on the subject?